frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailView.tsx

```diff
@@ -244,16 +242,20 @@ export function ObjectDetailView({
     [zoomedRowID, followForeignKey],
   );
 
-  const areImplicitActionsEnabled =
+  const areImplicitActionsEnabled = Boolean(
```

Comment:
The bug was that `areImplicitActionsEnabled` was `undefined` when `question` was `undefined` and then `useListActionsQuery` used `useEntityListQuery` which has a default value `enabled = true`.

Casting this to boolean is enough to fix the issue.